patches/chrome-android-java-src-org-chromium-chrome-browser-toolbar-top-ToolbarPhone.java.patch

```diff
@@ -0,0 +1,52 @@
+diff --git a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarPhone.java b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarPhone.java
+index 25c687987c90ac32f40698e9270cddc1d77f3bb0..0ea457c1b335d38d402d96ae559e792170854c57 100644
+--- a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarPhone.java
++++ b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarPhone.java
+@@ -89,7 +89,7 @@ import java.util.List;
+ /**
+  * Phone specific toolbar implementation.
+  */
+-public class ToolbarPhone extends ToolbarLayout implements Invalidator.Client, OnClickListener,
++public class ToolbarPhone extends BraveToolbarLayout implements Invalidator.Client, OnClickListener,
+                                                            NewTabPage.OnSearchBoxScrollListener,
+                                                            TabCountObserver {
+     /** The amount of time transitioning from one theme color to another should take in ms. */
+@@ -411,10 +411,12 @@ public class ToolbarPhone extends ToolbarLayout implements Invalidator.Client, O
+     /**
+      * Set the background color of the location bar to appropriately match the theme color.
+      */
+-    private void updateModernLocationBarColor(int color) {
```

Comment:
this can be done with asm

Reply:
Follow up issue created https://github.com/brave/brave-browser/issues/12322